serving/installing-logging-metrics-traces.md

```diff
@@ -1,83 +1,76 @@
-# Monitoring, Logging and Tracing Installation
+# Installing Logging, Metrics and Traces
 
-Knative Serving offers two different monitoring setups:
-[Elasticsearch, Kibana, Prometheus and Grafana](#elasticsearch-kibana-prometheus--grafana-setup)
-or
-[Stackdriver, Prometheus and Grafana](#stackdriver-prometheus--grafana-setup)
-You can install only one of these two setups and side-by-side installation of
-these two are not supported.
+If you installed the [full Knative release](../install/README.md#installing-knative),
+the monitoring components are already installed and you can skip down to the
```

Comment:
We should clarify what "monitoring components" actually get installed so that its clear what comes in that install bundle and what doesnt (ie. do they need to separately install metric or trace support).

Are "the monitoring components" that get installed with the .../serving/...../release.yaml still the "ELK stack with Prometheus and Grafana"?  Do you get zipkin too?